pytest_terraform/tf.py

```diff
@@ -152,10 +152,16 @@ class TerraformState(object):
     attributes which contain the key 'name' will also be present.
     """
 
-    def __init__(self, resources, outputs):
+    def __init__(self, resources, outputs, runner=None):
+        self._runner = runner
         self.outputs = outputs
         self.resources = resources
 
+    @property
+    def work_dir(self):
+        if self._runner:
+            return self._runner.work_dir
```

Comment:
Will this cause problems when runner isn't set? I'm unsure where working directory is typically invoked

Reply:
when the runner isn't set its typically due to being in replay mode in which case work_dir returns none, users of this would have to cognizant of that, by not using replay. the typical working directory is a subdir of the pytest tmpdirectory.